airflow-ctl/src/airflowctl/ctl/cli_config.py

```diff
@@ -530,16 +564,32 @@ def merge_commands(
 
 command_factory = CommandFactory()
 
+AUTH_ARGS = (ARG_AUTH_URL, ARG_AUTH_TOKEN, ARG_AUTH_ENVIRONMENT, ARG_AUTH_USERNAME, ARG_AUTH_PASSWORD)
+
 AUTH_COMMANDS = (
     ActionCommand(
         name="login",
         help="Login to the metadata database for personal usage. JWT Token must be provided via parameter.",
         description="Login to the metadata database",
         func=lazy_load_command("airflowctl.ctl.commands.auth_command.login"),
-        args=(ARG_AUTH_URL, ARG_AUTH_TOKEN, ARG_AUTH_ENVIRONMENT, ARG_AUTH_USERNAME, ARG_AUTH_PASSWORD),
```

Comment:
We can revert the auth changes regarding my below comment

Reply:
Done 👍🏻